Repositories/TagRepository.cs

```diff
@@ -0,0 +1,63 @@
+/*
+* Digital Excellence Copyright (C) 2020 Brend Smits
+* 
+* This program is free software: you can redistribute it and/or modify 
+* it under the terms of the GNU Lesser General Public License as published 
+* by the Free Software Foundation version 3 of the License.
+* 
+* This program is distributed in the hope that it will be useful, 
+* but WITHOUT ANY WARRANTY; without even the implied warranty 
+* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+* See the GNU Lesser General Public License for more details.
+* 
+* You can find a copy of the GNU Lesser General Public License 
+* along with this program, in the LICENSE.md file in the root project directory.
+* If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+*/
+
+using Microsoft.EntityFrameworkCore;
+using Models;
+using Repositories.Base;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Repositories
+{
+
+    /// <summary>
+    ///     This is the interface of the tag repository
+    /// </summary>
+    /// <seealso cref="IRepository{Tag}"/>
+    public interface ITagRepository : IRepository<Tag> {
+
+        /// <summary>
+        ///     This interface method gets all tags asynchronously
+        /// </summary>
+        /// <returns></returns>
+        Task<List<Tag>> GetAllAsync();
```

Comment:
Haven't given much thought to it to be honest. Since it uses ToListAsync it is also kind of a fake async.

The reason I did it is that I took inspiration from other repositories, which did the same thing. (RoleRepository)

Would you suggest removing it and using the one from the base class?

Reply:
Yes, I think that would be the best solution to keep everything DRY. Maybe the base method has been added later, but I can't say anything for sure about that.